### DISCUSSION TITLE
Add tag to resource equinix_ecx_l2_connection

### DISCUSSION THREAD

**Discussion by milko3666** [equinix/terraform-provider-equinix#330] [Ideas]
Hi community,
Nowaday we have a lot of resources that you can add tags, tags usually is used by controlling the cost, building blocks, etc. The resource "equinix_ecx_l2_connection" has not this feature, I think that it's interesting add this bit feature.

**Comment by ctreatma** [comment]
The ECX API has been shut down and related resources were removed from the Terraform provider in v2.0.0.
